src/routes/controllers/password.js

```diff
@@ -0,0 +1,92 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import bcrypt from 'bcryptjs';
+import models from '../../db/models';
+import sendMailer from '../../config/mailConfig';
+
+dotenv.config();
+
+const { User } = models;
+const secret = process.env.SECRET_KEY;
+const expires = { expiresIn: '2h' };
+const generateToken = payload => jwt.sign(payload, secret, expires);
+let message;
+const Password = {
+/**
+   * Collect recovery email
+   * @param {object} req
+   * @param {object} res
+   * @sends {mail} mail
+   * @returns {object} email
+   */
+  async forgotPassword(req, res) {
+    const { email } = req.body;
+
+    try {
+      const checkUser = await User.findOne({
```

Comment:
We can abstract this into a helper function that checks if user exists.

Reply:
I wrote this so that can my code could work, all reusable functions have been written by Favour and Dimkpa as utils, when their PR is merged I will rebase and include them in my code and  update my PR, then inform you

Reply:
this has been resolved